eng/pipelines/aggregate-reports.yml

```diff
@@ -104,13 +104,11 @@ jobs:
         GdnBreakAllTools: false
         GdnBreakGdnToolCredScan: true
         GdnBreakGdnToolCredScanSeverity: Error
-        # GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
-        # GdnBreakBaselines: baseline  
+        GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
+        GdnBreakBaselines: baseline  
         # Used for generating baseline file.
-        GdnBreakOutputBaselineFile: java
-        GdnBreakOutputBaseline: baseline
-      condition: succeededOrFailed()
-      continueOnError: true
+        # GdnBreakOutputBaselineFile: java
```

Comment:
What are these here commented out?

Reply:
The commented codes are used for generating the baseline file. Once we have the file, we only need to run the above ones